### DISCUSSION TITLE
Create Issued Document - 500 error on invoice creation

### DISCUSSION THREAD

**Discussion by Davide-T** [fattureincloud/api#22] [Q&A] [answered]
Hi, I have problems when I create a new invoice with api v2. I always get a 500 error. This is the body of the test request:

`{
    "data": {
        "type": "invoice",
        "entity": {
            "name": "Mario Rossi",
            "vat_number": "47803200154",
            "tax_code": "RSSMRA91M20B967Q",
            "address_street": "Via Italia, 66",
            "address_postal_code": "20900",
            "address_city": "Milano",
            "address_province": "MI",
            "address_extra": "",
            "country": "Italia"
        },
        "date": "2022-01-20",
        "number": 180,
        "numeration": "/fatt",
        "subject": "internal subject",
        "visible_subject": "visible subject",
        "currency": {
            "id": "EUR",
            "exchange_rate": "1.00000",
            "symbol": "€"
        },
        "language": {
            "code": "it",
            "name": "Italiano"
        },
        "items_list": [
            {
                "product_id": "default",
                "code": "default",
                "name": "test",
                "net_price": 0.0,
                "category": "cucina",
                "discount": 0,
                "qty": 1,
                "vat": {
                    "id": 0
                }
            }
        ],
        "payment_method":{
            "id": 386683
        },
        "payments_list": [
            {
                "amount": 0,
                "due_date": "2022-01-23",
                "paid_date": "2022-01-22",
                "status": "paid",
                "payment_account": {
                    "id": 808251
                }
            }
        ]
    }
}`

Thanks for your help

**Comment by emajo** [comment, marked as answer]
Hi @Davide-T, the problem is that the `payment_method` you are specifying does not exist in your company, you can see your payment methods making a [list payment methods](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/info/payment_methods) call.

We know that the 500 error is not the correct one and it's not enough specific and we are working on it,
thanks,
Emanuele

**Reply by Davide-T** [reply]
@emajo Thanks, now it works!
